rustls/src/server/tls13.rs

```diff
@@ -63,6 +63,13 @@ mod client_hello {
 
     use super::*;
 
+    #[derive(PartialEq)]
+    pub(super) enum EarlyData {
```

Comment:
Style nit: maybe name this `EarlyDataState`, since it doesn't contain any early data itself?